.npmignore

```diff
@@ -1,6 +1,12 @@
-node_modules/
-src/
-build/
+node_modules
+src
+build
 .arcconfig
-.babelrc
 *.yml
+.editorconfig
+karma.config.js
+travis.yml
+CHANGELOG.md
+CONTRIBUTING.md
+lib/test
```

Comment:
I believe we included files like .editorconfig and karma.config.js so that people can unit test the library as well as contribute changes.  Wouldn't this make it more difficult for them to do so?